handler/rfc8628/auth_handler_test.go

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package rfc8628_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	"github.com/ory/fosite/storage"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ory/fosite"
+	. "github.com/ory/fosite/handler/rfc8628"
```

Comment:
also.. why do this.. the general pattern is that we'd want to test the package as others see it..

Reply:
Not sure what you mean, are you referring to the import or to the whole test file? There are "end-to-end" tests in the `integration` dir.